### DISCUSSION TITLE
Table borders

### DISCUSSION THREAD

**Discussion by PeterParslow** [agiorguk/gemini#93] [General]
Page 1049 contains some tables; in the "old" live pages these have visible borders around each cell, and in the ASCIIdoc preview of the new pages, that's also true.
But by the time they get to HTML they have faint horizontal borders (between table rows) but no visible borders between table columns.

This may not matter much, because the text sits neatly in each column.

**Comment by archaeogeek** [comment]
I think this is coming from https://github.com/agiorguk/gemini/blob/main/docs/assets/gemini.css#L15 so it should definitely be possible to tweak the table css
